chain/db/table.go

```diff
@@ -44,7 +44,7 @@ type DataSourceRevision struct {
 	Name           string `gorm:"not null"`
 	Timestamp      int64  `gorm:"not null"`
 	BlockHeight    int64  `gorm:"not null"`
-	TxHash         []byte `gorm:"not null"`
+	TxHash         []byte `sql:"default:null"`
```

Comment:
I don't think we need to put in default?

Reply:
NULL should be the automatic default for table that is nullable, no?

Reply:
If not set this tag, I cannot add nil of []byte as null in table.